xero_files.yaml

```diff
@@ -77,6 +77,7 @@ paths:
       responses:
         '200':
           description: A successful request
+          x-isMultipartUpload: true
```

Comment:
This is also a new extension. I was having challenges segmenting only the `fileUpload` function and not impacting the accounting API using the `#isFile` flag or `x-isAttachment`.

`UploadFile` is the only endpoint out of our entire API sets that used the 'multi-part' http POST method. This ext made the logic much cleaner in the nod, but open to try to use existing spec if you have any tips.